Civi/Funding/Api4/Action/FundingCaseType/UpdateAction.php

```diff
@@ -49,76 +49,70 @@ public function _run(Result $result): void {
     $paymentInstructionTemplateFileId = $this->values['payment_instruction_template_file_id'] ?? NULL;
     unset($this->values['payment_instruction_template_file_id']);
 
+    $paybackClaimTemplateFileId = $this->values['payback_claim_template_file_id'] ?? NULL;
+    unset($this->values['payback_claim_template_file_id']);
+
     parent::_run($result);
 
     if (NULL !== $transferContractTemplateFileId) {
       $transferContractTemplateFileId = (int) $transferContractTemplateFileId;
-      $this->updateTransferContractTemplate($transferContractTemplateFileId);
+      $this->updateTemplate(
+        'transfer_contract_template_file_id',
+        $transferContractTemplateFileId,
+        FileTypeNames::TRANSFER_CONTRACT_TEMPLATE
+      );
```

Comment:
You mean the `\InvalidArgumentException`? It's actually just a safeguard. This can only happen for user generated calls (e.g. via API explorer) so I don't think we need to handle it.